### DISCUSSION TITLE
The new version does not collect ifHCInOctets

### DISCUSSION THREAD

**Discussion by hufhend** [prometheus/snmp_exporter#963] [Q&A]
### Host operating system: output of `uname -a`
docker: Linux d369f3590257 5.16.17-sun50iw9 https://github.com/prometheus/snmp_exporter/pull/3.0.6 SMP Tue Aug 9 13:51:16 CST 2022 aarch64 GNU/Linux

### snmp_exporter version: output of `snmp_exporter -version`

snmp_exporter, version 0.23.0 (branch: HEAD, revision: https://github.com/prometheus/snmp_exporter/commit/20657d2f486a1ac237238827beed97fa67eb8611)
build user: root@e4de29736f81
build date: 20230816-09:52:12
go version: go1.20.7
platform: linux/arm64
tags: netgo

### What device/snmpwalk OID are you using?
Mikrotik

### What did you do that produced an error?
I promoted to the new version, or rather the promotion was done by crontab

### What did you expect to see?
I expected it to go further

### What did you see instead?
I still have the old snmp-exporter and some new ones. I checked the old and new configuration files and they contain **ifHCInOctets in the Mikrotik** section. But the metrics on the new one do not contain any ifHCInOctets lines, while the old one does. 
The same goes for ifHCOutOctets or ifPhysAddress.

For the new one, I use the docker default new configuration file in one case and in the other I have a newly ungenerated one. Unfortunately with the new one it doesn't work for either.

**Comment by SuperQ** [comment]
We intentionally removed duplication of IF-MIB from vendor-specific sections. The default snmp.yml is meant as an example, not a canonical config.

**Reply by hufhend** [reply]
What are we supposed to do about it now?

**Reply by SuperQ** [reply]
* Generate your own custom snmp.yml.
* Setup multiple scrape jobs.
* Wait for [muti-module scrape support](https://github.com/prometheus/snmp_exporter/pull/945)

**Reply by hufhend** [reply]
I'm sorry, there was an effort, I even redid the Grafana dashboards, but the data that used to be there is not there. For example, brigde is all zeros. 
I'm going back to v0.22

**Reply by SuperQ** [reply]
The multi-module scrape support is now released.

**Reply by hufhend** [reply]
Thanks, I'll create a test OrangePi and try it there, until then I'll hold production at v0.22. 
There are just too many changes at once.

**Reply by klzsysy** [reply]
I also went back to v0.22, I'm not an SNMP expert, I just want it to work out of the box
